streets_poles/init.lua

```diff
@@ -39,7 +39,9 @@ minetest.register_node(":streets:bigpole", {
 		[3] = { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0, 0 },
 		["t"] = { 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1 },
 		["b"] = { 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1 }
-	}
+	},
+	check_for_pole = true
```

Comment:
This should be explained so that the next modder knows what the purpose is.
```suggestion
	check_for_pole = true -- allows street_signs to use this as a pole to attach signs
```